src/protocol.rs

```diff
@@ -561,9 +462,13 @@ mod tests {
         }
         Ok(())
     }
+    /*
     /// `presign_identifier` identifies a sub-session for pre-signing and
     /// signing.
-    fn is_presigning_done(quorum: &[Participant], presign_identifier: Identifier) -> Result<bool> {
+    fn is_presigning_done(
```

Comment:
note: commented these out because of aforementioned rabbit hole, but they've been replaced by just checking that all participants have returned output.